src/backend/app.js

```diff
@@ -1,30 +1,36 @@
 const express = require('express');
 const app = express();
-// const fs = require("fs");
+const fs = require("fs");
 const tvShows = require("./ShowsList");
 const path = require('path');
 const imageRes = require('./image/ResolveID');
 const port = process.env.PORT || 3001;
 
+//downloading movie data
 app.get('/rest/show', (req, res)=>{
-    
+   console.log('rest api requested'); 
```

Comment:
Can you please remove the `console.log`? They shouldn't be on prod code unless you have a very good reason (e.g. if you are trying to debug an issue in prod)